packages/framework/src/Resources/config/parameters_common.yaml

```diff
@@ -33,3 +33,9 @@ parameters:
         - 'admin_login'
         - 'admin_login_check'
         - 'admin_logout'
+
+    shopsys.admin_context_route_prefixes:
```

Comment:
I would be slightly happier if we used configuration instead of parameter (see \Shopsys\FrameworkBundle\DependencyInjection\Configuration), but I can live with this solution